### PR TITLE
Add host Playwright to support linux arm64 download

### PR DESCRIPTION
**Pull request by knowlet**
# Development guide

[Link](https://github.com/go-rod/rod/blob/master/.github/CONTRIBUTING.md)

## Test on local before making the PR

```bash
go run ./lib/utils/simple-check
```

As I mentioned in https://github.com/go-rod/rod/issues/586, by adding Playwright host can download the prebuilt binaries for linux arm64.

### CODE REVIEW

**Review by ysmood — CHANGES_REQUESTED** [review]
Please resolve the conflicts

File: lib/launcher/revision.go [hunk 1]
```diff
@@ -3,4 +3,4 @@
 package launcher
 
 // DefaultRevision for chromium
-const DefaultRevision = 993261
+const DefaultRevision = 983172
```
Comment (ysmood) [inline lib/launcher/revision.go:6]: How did you find out this revision works?

Comment (knowlet) [reply lib/launcher/revision.go:6]: Revision is based on the [`BUILD_NUMBER`](https://github.com/microsoft/playwright/blob/main/browser_patches/chromium/BUILD_NUMBER) in Playwright's repo and the zipName is in [`EXPECTED_BUILDS`](https://github.com/microsoft/playwright/blob/main/browser_patches/chromium/EXPECTED_BUILDS), but after chromium 100 the number is not the same as the chromium revision in Google snapshots and NPM mirror, or it's better to follow Playwright's build version in another way?

**Comment by ysmood** [comment]
https://github.com/go-rod/rod/commit/f08e9658e7e8d0b1cb8d883d62ff1ae7aad4a0c8

**Pull request closed without merging**